pages/index.tsx

```diff
@@ -13,6 +21,48 @@ export default function Home() {
       <section className={styles.container}>
         <main className={styles.MainArea}>
           <HeroBanner imagePath='logo.svg' altText='TCSCA Logo' />
+          {/* NOTE: we may want to put stuff in seperate sections rather than back to back */}
+          {homeSections.map((section, index) => {
+            const sectionStyle =
```

Comment:
maybe we just use an `oddSection` and `evenSection` class it might just make things a bit simplier.

Alternatively maybe we just use:
```css
:nth
```

Reply:
Did it in an alternative way